boards/nordic/bm_nrf54l15dk/bm_nrf54l15dk_nrf54l05_cpuapp_s115_softdevice.dts

```diff
@@ -59,23 +59,18 @@
 &cpuapp_sram {
 	status = "okay";
 
-	partitions {
-		#address-cells = <1>;
-		#size-cells = <1>;
-
-		softdevice_static_ram: partition@20000080 {
-			label = "softdevice_static_ram";
-			reg = <0x20000080 0x1380>;
-		};
+	softdevice_static_ram: sram@0 {
+		label = "softdevice_static_ram";
+		reg = <0x0 0x1380>;
```

Comment:
Please double check that this is correct, the original is `reg = <0x20000080 0x1380>;`.
Should the new value be `reg = <0x80 0x1380>;` instead?

Reply:
The files are correct: https://github.com/nrfconnect/sdk-nrf-bm/blob/main/boards/nordic/bm_nrf54l15dk/bm_nrf54l15dk_nrf54l15_cpuapp_common.dtsi#L35